src/model/Me.ts

```diff
@@ -27,6 +27,11 @@ export type PhoneVerificationResponse = {
   verify_phone: boolean;
 };
 
+export type KYCVerificationResponse = {
+  state: boolean;
+  type: "credit-card" | "phone" | "ticket" | null;
```

Comment:
Does this have to be exactly null or it can be undefined? if is just an optional field we can have it as `type?: ...`

Reply:
it is exactly `null`